library/src/test/java/com/ironz/binaryprefs/encryption/XorKeyEncryptionImplTest.java

```diff
@@ -0,0 +1,35 @@
+package com.ironz.binaryprefs.encryption;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * @author Sergey Boishtyan
+ */
+public class XorKeyEncryptionImplTest {
```

Comment:
Also you should check that xor'ed byte array will be incorrect de-xor'ed with another xor byte array. Just watch on `AesByteEncryptionImpl.java`.